docs/products/kafka/howto/prevent-full-disks.rst

```diff
@@ -0,0 +1,62 @@
+Prevent full disks
+===================
+
+To prevent **Aiven for Apache Kafka** from malfunctioning, the Aiven platform detects when the available disk space in **Aiven for Apache Kafka** services is running low and when the usage exceeds 90% notifies the customer.
```

Comment:
```suggestion
To prevent **Aiven for Apache Kafka** from malfunctioning, the Aiven platform detects when the available disk space in **Aiven for Apache Kafka** services is running low, and when the usage exceeds 90% notifies the customer.
```